cmd/lakectl/cmd/log.go

```diff
@@ -17,9 +17,10 @@ Date:          {{ $val.CreationDate|date }}
 Merge:         {{ $val.Parents|join ", "|bold }}
 {{ end }}
 	{{ $val.Message }}
-	
+
+Metadata:
 	{{ range $key, $value := $val.Metadata.AdditionalProperties }}
-		{{ $key }} = {{ $value }}
+	{{ $key }}	=	{{ $value }}
```

Comment:
I actually like this form less, it is harder to understand what is going on.

Options:
* Have JSON output format (for all of lakectl...) for people who want to know what's really going on.
* Put metadata in a table, I think we have enough support in our formatting for that.
* Try to align keys but don't worry if that doesn't work.

  Something like this might be good enough:
  ```suggestion
  	{{ $key | printf "%-18s" }} = {{ $value }}
  ```